app/views/CanYouMakeAnUpfrontPayment.scala.html

```diff
@@ -85,7 +86,7 @@
                     )
                 ),
                 classes = "govuk-radios--inline",
-                errorMessage = form.error(key).map(e => ErrorMessage(content = Text(Messages.UpfrontPayment.getError(e.message).show)))
+                errorMessage = form.error(key).map(e => ErrorMessage(content = Text(e.message)))
```

Comment:
Again, here a `e.message` contains localised message which was defined in form mapping.